Log elapsed time between request and response

0xced opened the pull request.

0xced committed:
Log elapsed time between request and response

mattt commented:
Awesome work. I was considering adding start / end date to `AFURLConnectionOperation` for just this very same purpose. Would be interested to hear your thoughts on that.

kevinbarrett commented:
@mattt I'm looking for a way to figure out total elapsed time in a connection operation, and start/end dates sound perfect.

0xced commented:
This is quite funny because I initially added an `elapsedTime` property on `AFURLConnectionOperation`. Then I remembered your talk _What I Learned From AFNetworking's GitHub Issues_ and I thought that would be something you would reject! So I deleted the `elapsedTime` property and I added the feature where it is actually needed, i.e. in `AFHTTPRequestOperationLogger`.

I needed the elapsed time somewhere else, so I just implemented it like this:

``` objective-c
NSDate *startDate = [NSDate date];
[client getPath:path parameters:parameters success:^(AFHTTPRequestOperation *operation, id responseObject) {
    NSTimeInterval elapsedTime = [[NSDate date] timeIntervalSinceDate:startDate];
    [self.refreshControl performSelector:@selector(endRefreshing) withObject:nil afterDelay:fmaxf(0.f, 0.4f - elapsedTime)];
} failure:nil];
```

I think this is simple enough to code to not require a new property on `AFURLConnectionOperation`.
